server/scala/project/BuildSupport.scala

```diff
@@ -151,9 +153,17 @@ object BuildSupport {
     }
 
   lazy val mapping: List[(String, String)] = {
+    println(libdir)
     val libFileList =
-      new java.io.File(libdir.substring(3)).listFiles
+      new java.io.File(libdir).listFiles
         .filter(_.isFile)
+        .filter { x =>
+          println(x)
+          // only want the lib files with a single period, for the filename like .dylib, .so, .dll.
+          x.getName
```

Comment:
So getName is the cross-platform method of just getting the file name minus the path?